tests/integration/tasks/test_epicor.py

```diff
@@ -0,0 +1,20 @@
+from viadot.tasks import EpicorOrdersToDF
+from viadot.config import local_config
+
+
+def test_epicor_orders_to_df():
+    task = EpicorOrdersToDF(
+        config_key="Epicor",
```

Comment:
I would use all caps as for all other cred keys (except DuckDB, not sure what happened there :D)